// @expo-google-fonts/roboto-condensed/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const RobotoCondensed_100Thin = require('./100Thin/RobotoCondensed_100Thin.ttf');
export const RobotoCondensed_200ExtraLight = require('./200ExtraLight/RobotoCondensed_200ExtraLight.ttf');
export const RobotoCondensed_300Light = require('./300Light/RobotoCondensed_300Light.ttf');
export const RobotoCondensed_400Regular = require('./400Regular/RobotoCondensed_400Regular.ttf');
export const RobotoCondensed_500Medium = require('./500Medium/RobotoCondensed_500Medium.ttf');
export const RobotoCondensed_600SemiBold = require('./600SemiBold/RobotoCondensed_600SemiBold.ttf');
export const RobotoCondensed_700Bold = require('./700Bold/RobotoCondensed_700Bold.ttf');
export const RobotoCondensed_800ExtraBold = require('./800ExtraBold/RobotoCondensed_800ExtraBold.ttf');
export const RobotoCondensed_900Black = require('./900Black/RobotoCondensed_900Black.ttf');
export const RobotoCondensed_100Thin_Italic = require('./100Thin_Italic/RobotoCondensed_100Thin_Italic.ttf');
export const RobotoCondensed_200ExtraLight_Italic = require('./200ExtraLight_Italic/RobotoCondensed_200ExtraLight_Italic.ttf');
export const RobotoCondensed_300Light_Italic = require('./300Light_Italic/RobotoCondensed_300Light_Italic.ttf');
export const RobotoCondensed_400Regular_Italic = require('./400Regular_Italic/RobotoCondensed_400Regular_Italic.ttf');
export const RobotoCondensed_500Medium_Italic = require('./500Medium_Italic/RobotoCondensed_500Medium_Italic.ttf');
export const RobotoCondensed_600SemiBold_Italic = require('./600SemiBold_Italic/RobotoCondensed_600SemiBold_Italic.ttf');
export const RobotoCondensed_700Bold_Italic = require('./700Bold_Italic/RobotoCondensed_700Bold_Italic.ttf');
export const RobotoCondensed_800ExtraBold_Italic = require('./800ExtraBold_Italic/RobotoCondensed_800ExtraBold_Italic.ttf');
export const RobotoCondensed_900Black_Italic = require('./900Black_Italic/RobotoCondensed_900Black_Italic.ttf');
