// @expo-google-fonts/fraunces/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const Fraunces_100Thin = require('./100Thin/Fraunces_100Thin.ttf');
export const Fraunces_200ExtraLight = require('./200ExtraLight/Fraunces_200ExtraLight.ttf');
export const Fraunces_300Light = require('./300Light/Fraunces_300Light.ttf');
export const Fraunces_400Regular = require('./400Regular/Fraunces_400Regular.ttf');
export const Fraunces_500Medium = require('./500Medium/Fraunces_500Medium.ttf');
export const Fraunces_600SemiBold = require('./600SemiBold/Fraunces_600SemiBold.ttf');
export const Fraunces_700Bold = require('./700Bold/Fraunces_700Bold.ttf');
export const Fraunces_800ExtraBold = require('./800ExtraBold/Fraunces_800ExtraBold.ttf');
export const Fraunces_900Black = require('./900Black/Fraunces_900Black.ttf');
export const Fraunces_100Thin_Italic = require('./100Thin_Italic/Fraunces_100Thin_Italic.ttf');
export const Fraunces_200ExtraLight_Italic = require('./200ExtraLight_Italic/Fraunces_200ExtraLight_Italic.ttf');
export const Fraunces_300Light_Italic = require('./300Light_Italic/Fraunces_300Light_Italic.ttf');
export const Fraunces_400Regular_Italic = require('./400Regular_Italic/Fraunces_400Regular_Italic.ttf');
export const Fraunces_500Medium_Italic = require('./500Medium_Italic/Fraunces_500Medium_Italic.ttf');
export const Fraunces_600SemiBold_Italic = require('./600SemiBold_Italic/Fraunces_600SemiBold_Italic.ttf');
export const Fraunces_700Bold_Italic = require('./700Bold_Italic/Fraunces_700Bold_Italic.ttf');
export const Fraunces_800ExtraBold_Italic = require('./800ExtraBold_Italic/Fraunces_800ExtraBold_Italic.ttf');
export const Fraunces_900Black_Italic = require('./900Black_Italic/Fraunces_900Black_Italic.ttf');
